governing-documents/project-annual-review.md

```diff
@@ -10,81 +10,89 @@ nav_order: 5
 # Project Annual Review
 
 ## Overview
+
 In addition to the [quarterly project updates](./project-updates.md), projects will undergo an annual review by the TAC. This annual review will replace one of the quarterly reports. Unlike the quarterly updates, the annual review process will be a "big picture" assessment of the project by reviewing the progress on goals from the prior year and goals for the next year. In addition, the annual review process will include a deep dive into the project health, including contributor and maintainer diversity and adoption metrics, and will include an assessment of the current [project lifecycle state](./project-lifecycle.md). The review will result in a set of recommendations for the project to improve and/or recommendation to move a project to another stage.
 
 ## Timeline
-Annual reviews will replace the Q1 quarterly report.  **If your annual review is not submitted within two months of notification, we will take this as a sign that the project is not under active maintenance and the TAC is likely to decide to archive the project and move it to End of Life status.**
 
-**NOTE:** If a project has genuinely stalled we can save everyone’s time and effort by archiving the project and moving it to `End of Life`.
+Annual reviews will replace the Q1 quarterly report.  **If a project's annual review is not submitted within two months of notification, we will take this as a sign that the project is not under active maintenance and the TAC is likely to decide to archive the project and move it to End of Life status.**
+
+**NOTE:** If a project has genuinely stalled the project's TSC can save everyone’s time and effort by archiving the project and moving it to `End of Life`.
 
 ## Process
+
 The process for annual reviews is as follows:
 
-* A TAC representative will be chosen on a round-robin basis to lead the review once the project files a PR. This representative should be someone that is not directly involved with the project.
-* A secondary representative will be chosen on a round-robin basis to observe and corroborate the findings of the lead. This representative should be someone that is not directly involved with the project.
-* The lead TAC member reviews the content of the PR and analyzes the project for community health indicators, their findings are placed within a thread in the TAC channel ([#toc](https://discord.com/channels/905194001349627914/941384040316018790)) for discussion. The thread should contain:
-    1. important facts about the project that could influence the TACs decision around the future of the project, its current stage, and path to other stages.
```

Comment:
I notice that these sentences have been removed. It seems necessary to establish guidance that suggests the review should be objective and holistic, considering all aspects such as encouraging projects to plan for the next stage in their lifecycle, as well as ensuring ongoing viability.